examples/connectivity/plot_extract_regions_canica_maps.py

```diff
@@ -0,0 +1,92 @@
+"""
+Regions extraction using Canonical ICA maps
+===========================================
+
+This example specifically shows how to segment each ICA map (a 4D Nifti
+image/object) into a distinct seperated brain region and extracts timeseries
+signals from each seperated region. Both can be done at the same time
+using `RegionExtractor`.
+
+We used 40 resting state functional datasets and `CanICA` for 4D components.
+
+This example motivates, how to use `RegionExtractor` to study
+functional connectomes using correlation and partial correlation
+matrices.
+Please see the related documentation of `RegionExtractor` for more details.
+"""
+import numpy as np
+from nilearn import datasets
+print (" -- Fetching ADHD resting state functional datasets -- ")
+adhd_dataset = datasets.fetch_adhd(n_subjects=20)
+func_filenames = adhd_dataset.func
+confounds = adhd_dataset.confounds
+
+from nilearn.input_data import NiftiMasker
+print (" -- Computing the mask from the data -- ")
+func_filename = func_filenames[0]
+masker = NiftiMasker(standardize=False, mask_strategy='epi')
```

Comment:
Do not use a masker, especially if you estimate the mask on only one subject. Let CanICA do it.